extensions/wikia/VisualEditorTourExperiment/scripts/VisualEditorTourExperimentInit.js

```diff
@@ -8,20 +8,21 @@ define('VisualEditorTourExperimentInit',
 			usersWithoutEditExperimentId = 5735670451;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the declaration of the unused 'usersWithoutEditExperimentId' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "usersWithoutEditExperimentId" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS1854)